tox.ini

```diff
@@ -21,6 +21,7 @@ deps =
     codecov
     freezegun
     psycopg2-binary==2.8.6
+    django-polymorphic
```

Comment:
Do we need `django-polymorphic` for testing? isn't it possible to add test without this?

Reply:
yes. it's happening with a Polymorphic model so need this dependency else i wont be able to reproduce.

Reply:
I would take a look. it may be possible to reproduce the problem without `Polymorphic`. Otherwise, we can accept it.

Reply:
Ok. thanks. 
this is currently became a major blocker for my project.

Reply:
btw, test still failing

Reply:
where is this rdoc?
`pip install -rdocs/requirements.txt`

Reply:
Thanks @rahulpr27git.

Seems hard to reproduce the problem without `Polymorphic`. I would suggest just keeping the change and reverting all the changes related to the test. we will accept this change without tests. Because the fix itself is clear and I don't want to include another project just for one single test.

Reply:
@hramezani please check again.

Reply:
Thanks @rahulpr27git